Cargo.toml

```diff
@@ -10,6 +10,10 @@ license = "MIT OR Apache-2.0 OR Zlib"
 owo-colors = "3.5.0"
 winit = { version = "0.28.6", default-features = false }
 
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+wasm-bindgen-test = "0.3"
+web-time = "0.2"
```

Comment:
I'm using [`web-time`](https://crates.io/crates/web-time) (my own crate) here instead of [`instant`](https://crates.io/crates/instant), because `instant` is unmaintained and has a bunch of issues.